src/Image.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import classNames from 'classnames';
+
+const Image = React.createClass({
+
+  propTypes: {
+    src: React.PropTypes.string,
+    alt: React.PropTypes.string,
+    responsive: React.PropTypes.bool,
+    rounded: React.PropTypes.bool,
+    circle: React.PropTypes.bool,
+    thumbnail: React.PropTypes.bool
+  },
+
+  getDefaultProps() {
```

Comment:
Thank you @taion for the review. Does it mean the getDefaultProps() method is not needed here?